### PR TITLE
LambdaTest Taiko-Gauge Sample

### PR DESCRIPTION
**Pull request by mukesh-lt**
Build id 10808873

### CODE REVIEW

**Review by asad9711 — APPROVED** [review]

**Review by ankur-lt — CHANGES_REQUESTED** [review]
Will be releasing new version of plugin. We will add those changes into this PR too.

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-Lambdatest Taiko Sample
+# Lambdatest Taiko Sample
+
+## Pre-requisite
+- Node should be installed
+- Clone the repo using `git clone git@github.com:LambdaTest/lambdatest-taiko-sample.git`
+- Move to taiko-gauge folder using `cd lambdatest-taiko-sample/taiko-gauge`
+- Install taiko globally using `npm install -g taiko`
```
Comment (ankur-lt) [inline README.md]: Just npm install will work as all these dependencies are already added in package.json

File: taiko-gauge/tests/step_implementation.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    text,
+    into,
+    textBox,
+    evaluate
+} = require('taiko');
+const assert = require("assert");
+const headless = process.env.headless_chrome.toLowerCase() === 'true';
+
+const capabilities = {
+    'browserName': 'Chrome',
```
Comment (ankur-lt) [inline taiko-gauge/tests/step_implementation.js:26]: Also include other caps given here: https://github.com/LambdaTest/playwright-sample/blob/2801895c85e754580816af0dace3c849cceedc99/playwright-single.js#L8-L21

**Review by ankur-lt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ankur-lt) [reply README.md]: This needs to be modified.

**Review by ankur-lt — CHANGES_REQUESTED** [review]

**Review by ankur-lt — APPROVED** [review]

**Pull request merged**